pepper-apis/src/main/java/org/broadinstitute/ddp/export/collectors/PicklistQuestionFormatStrategy.java

```diff
@@ -113,14 +104,6 @@
         List<String> headers = new ArrayList<>();
         headers.add(definition.getStableId());
 
-        for (PicklistGroupDef group : definition.getGroups()) {
```

Comment:
group options are already included in definition.getAllPicklistOptions()
deleted to avoid duplicates